pages/common/keepass2.md

```diff
@@ -0,0 +1,15 @@
+# keepass2
+
+> A free, open source, light-weight and easy-to-use password manager.
```

Comment:
FOSS is arguably irrelevant (most of the commands we cover are); "light-weight" is acceptable; but "easy-to-use" is too subjective and opinionated. I'd suggest going with a plain, objective description -- `A light-weight password manager` would work.